dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/impl/PostgreSQLHelper.java

```diff
@@ -31,23 +31,52 @@
     
     private static final String PROP_DEFAULT_FETCH_SIZE = "DefaultFetchSize";
     private static final String PROP_PREPARE_THRESHOLD = "PrepareThreshold";
+    private static final String PROP_AUTOSAVE = "AutoSave";
 
     @SuppressWarnings("deprecation")
     private transient com.ibm.ejs.ras.TraceComponent jdbcTC = com.ibm.ejs.ras.Tr.register("com.ibm.ws.postgresql.logwriter", "WAS.database", null);
 
     private transient PrintWriter jdbcTraceWriter;
+    
+    private Class<?> Autosave;
```

Comment:
should this be volatile based on how it is used (lazy init)?

Reply:
it could be, yes. Since the cost of initializing it isn't very significant and double-computing the value isn't any difference in behavior I hadn't used it here.